locales/de.json

```diff
@@ -276,7 +276,8 @@
       "triggermethods": "Trigger Methoden",
       "deleteLanguage": "Eine Sprache löschen",
       "required": "Mit * markierte Felder sind auszufüllen.",
-      "updates": "Es ist empfohlen, demn Terminal.ink Discordserver beizutreten in dem du auf 'Discord' im Footer klickst, um Service updates zu erhalten, den aktuellen Verifizierungs-Status deines Bots zu sehen und um direkte Nachrichten vom Team über deine Applikation zu kriegen."
+      "updates": "Es ist empfohlen, demn Terminal.ink Discordserver beizutreten in dem du auf 'Discord' im Footer klickst, um Service updates zu erhalten, den aktuellen Verifizierungs-Status deines Bots zu sehen und um direkte Nachrichten vom Team über deine Applikation zu kriegen.",
```

Comment:
The note about the "Discord" button in the footer can get deleted now, it's right next to the text now